### PR TITLE
Wrap multiple retry blocks in the same one

### PR DESCRIPTION
**Pull request by nan-yu**

### CODE REVIEW

File: e2e/testcases/resource_conditions_test.go [hunk 1]
```diff
@@ -179,40 +175,38 @@ func TestResourceConditionAnnotations(t *testing.T) {
...
-	if err1 != nil || err2 != nil || err3 != nil {
-		if err1 != nil {
-			nt.T.Error(err1)
-		}
-		if err2 != nil {
-			nt.T.Error(err2)
-		}
-		if err3 != nil {
-			nt.T.Error(err3)
+		if err1 != nil || err2 != nil || err3 != nil {
```
Comment (karlkfi) [inline e2e/testcases/resource_conditions_test.go:195]: I think this if statement is redundant. But I suspect a MultiError would be better anyway, if we're going to wait for all of them to Validate before returning.

Comment (nan-yu) [reply e2e/testcases/resource_conditions_test.go]: DONE. Switched to use MultiError.

**Review by karlkfi — APPROVED** [review]
/lgtm

Much cleaner, thank you!

**Pull request merged**